proxy_test.go

```diff
@@ -260,7 +260,7 @@ func TestProxy(t *testing.T) {
 
 	grp.Go(func() error {
 		wg1.Wait() // sockets ready
-		proxy = zmq4.NewProxy(ctx, front, back, capt)
+		proxy, _ = zmq4.NewProxy(ctx, front, back, capt)
```

Comment:
we should probably check for that error.

Reply:
Done.